app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
+            $log->user_id = $user->id;
+            $log->severity = '5';
+            $log->category = 'mensa';
+            $log->text = json_encode((object) [
```

Comment:
Hmm, this feels wrong. this format isn't enforced anywhere, how do we interpret this for example in the frontend?